### PR TITLE
README: delocalise the AMO URL, other suggestions

### PR DESCRIPTION
**Pull request by grahamperrin**
Non-localised https://addons.mozilla.org/addon/reddituntranslate/ is better for an extension such as this.

Whilst here: a few other suggestions.

### CODE REVIEW

**Comment by grahamperrin** [comment]
Please note, I have not needed to use the extension (yet). 

I assume that _direction_ might be truer than _redirection_. In other words, I assume that a URL is de-localised **before** an unwanted visit to its localised equivalent.

**Comment by grahamperrin** [comment]
Side note: the localised `fr/firefox/` part of the link in <https://old.reddit.com/r/bugs/comments/1dfnsub/how_do_i_stop_the_automatic_translation_of_all/lkjbn2h/?context=3> …

**Comment by SeidSmatti** [comment]
Thanks a lot for your contribution, I aggree with everything except the redirection part, the extension detects the localised url and redirects the user to the localised one, but it's a matter of interpretation here

> Please note, I have not needed to use the extension (yet).
> 
> I assume that _direction_ might be truer than _redirection_. In other words, I assume that a URL is de-localised **before** an unwanted visit to its localised equivalent.

**Review by SeidSmatti — CHANGES_REQUESTED** [review]
I would still put redirection here, but thanks a lot for all the rest

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 ## Overview
 
-**redditUntranslate** is a Chrome extension designed to remove language parameters automatically added to Reddit URLs, ensuring that users are redirected to the original, untranslated version of the site. This extension is particularly useful for users who prefer browsing Reddit in its default language without any automatic translations.
+**redditUntranslate** removes language parameters that are automatically added to Reddit URLs, ensuring direction to original, non-translated content. This extension is particularly useful for users who prefer browsing Reddit in their default language without automatic translation.
```
Comment (grahamperrin) [inline README.md:6]:
```suggestion
**redditUntranslate** removes language parameters that are automatically added to Reddit URLs, ensuring redirection to original, non-translated content. This extension is particularly useful for users who prefer browsing Reddit in their default language without automatic translation.
```

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ The extension should now be installed temporarily. Note that it will be removed
 
 ## Usage
 
-Once installed, the Reddit Untranslate extension works automatically. Simply navigate to any Reddit URL, and the extension will remove any language parameters, redirecting you to the original version of the site.
+Once installed, redditUntranslate works automatically. Simply browse to any Reddit URL, the extension will remove any language parameters, directing you to the original version of the site.
```
Comment (grahamperrin) [inline README.md:48]:
```suggestion
Once installed, redditUntranslate works automatically. Simply browse to any Reddit URL, the extension will remove any language parameters, redirecting you to the original version of the site.
```

**Comment by grahamperrin** [comment]
Thank you!

**Pull request merged**